deploy-service/common/src/main/java/com/pinterest/deployservice/metrics/MeterConstants.java

```diff
@@ -0,0 +1,11 @@
+package com.pinterest.deployservice.metrics;
+
+import static com.pinterest.teletraan.universal.metrics.micrometer.PinStatsNamingConvention.CUSTOM_NAME_PREFIX;
+
+public class MeterConstants {
+  public final static String ERROR_BUDGET_METRIC_NAME = CUSTOM_NAME_PREFIX + "error-budget.counters";
+  public final static String ERROR_BUDGET_TAG_NAME_METHOD_NAME = CUSTOM_NAME_PREFIX + "method_name";
```

Comment:
Do you need `CUSTOM_NAME_PREFIX` for tag names?

Reply:
😱 we do not... my bad... I'll fix it.
Thanks for pointing that out.